nixos/modules/misc/ids.nix

```diff
@@ -346,6 +346,7 @@ in
       paperless = 315;
       #mailman = 316;  # removed 2019-08-30
       zigbee2mqtt = 317;
+      acme = 318;
```

Comment:
Do we need those to be fixed? Is not possible to just allocate a user?

Reply:
Hm, I guess not actually? I did this assuming it was good practice but there's no requirement for it to be fixed from an implementation POV especially with the acme-fixperms.service